style.css

```diff
@@ -0,0 +1,101 @@
+@import url('https://fonts.googleapis.com/css2?family=Poppins:wght@200;300;400;500;600;700&display=swap');
+
+.container {
+  background-color: #fffdfd;
+  margin: 0 auto;
+  padding: 0;
+  width: 375px;
+  height: 700px;
+  flex-shrink: 3;
+}
+
+header {
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+  margin-bottom: 10px;
+}
+
+header nav {
+  display: flex;
+  justify-content: center;
+}
+
+header a {
+  padding: 10px;
+  text-decoration: none;
+  font-family: 'Poppins', sans-serif;
+  font-style: normal;
+  font-weight: 700;
+  font-size: 18px;
+  line-height: 20px;
+  color: #6070ff;
+  margin-left: 10px;
```

Comment:
[OPTIONAL]
- [ ] Please make your website is responsive, following the breakpoints as required in the project requirement
![Screenshot from 2023-03-24 03-10-02](https://user-images.githubusercontent.com/91142432/227569493-7bce1966-d095-45c5-92b8-f4abfc766f58.png)
- Please use responsive values in your CSS rules, like percentages (i.e. width: 80%), instead of pixels to make your website responsive.